home_task_1/src/App.css

```diff
@@ -0,0 +1,89 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.title {
+  color: blue;
+}
+
+.counter {
+  color: black;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.content-btn {
+  display: flex;
+  justify-content: space-around;
+  margin: 10px 0;
+  width: 400px;
+}
+
+.btn-increment {
+  background-color: green;
+  border: none;
+  border-radius: 10px;
+  color: white;
+  cursor: pointer;
+  padding: 10px 25px;
+  font-size: 16px;
+}
+
+.btn-decrement {
+  background-color: red;
+  border: none;
+  border-radius: 10px;
+  color: white;
+  cursor: pointer;
+  padding: 10px 25px;
+  font-size: 16px;
+}
+.content-search {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  width: 800px;
+}
+input[type=text], select {
+  width: 150px;
+  padding: 12px 20px;
+  margin: 8px 0;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+}
+
+.box {
+  background-color: #ccc;
+  border-radius: 10px;
+  margin-bottom: 5px;
+  padding: 0 5px;
+  text-align: justify;
+  width: 550px;
+}
+
+.title-company {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.content-toggle {
+  width: 500px;
+}
+
+.genre {
+  width: 30%;
+  padding: 12px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.selected {
+  color: blueviolet;
+  font-size: 15px;
+}
```

Comment:
1. Split this file into smaller ones. 
Keep component related styles close to the component like this:
```
components/
     CreateElement/
           Counter.jsx
           Counter.scss
```

3. Start using `scss` as it most likely you won't meet plain `.css` in real-life project

Reply:
Implementing scss